core/local/change.js

```diff
@@ -88,24 +67,6 @@ function build (type /*: string */, path /*: string */, opts /*: ?{stats?: fs.St
   return change
 }
 
-function maybeAddFile (a /*: ?LocalChange */) /*: ?LocalFileAddition */ { return (a && a.type === 'FileAddition') ? a : null }
-function maybePutFolder (a /*: ?LocalChange */) /*: ?LocalDirAddition */ { return (a && a.type === 'DirAddition') ? a : null }
-function maybeMoveFile (a /*: ?LocalChange */) /*: ?LocalFileMove */ { return (a && a.type === 'FileMove') ? a : null }
-function maybeMoveFolder (a /*: ?LocalChange */) /*: ?LocalDirMove */ { return (a && a.type === 'DirMove') ? a : null }
-function maybeDeleteFile (a /*: ?LocalChange */) /*: ?LocalFileDeletion */ { return (a && a.type === 'FileDeletion') ? a : null }
-function maybeDeleteFolder (a /*: ?LocalChange */) /*: ?LocalDirDeletion */ { return (a && a.type === 'DirDeletion') ? a : null }
-
-function find /*:: <T> */ (changes /*: LocalChange[] */, maybeRightType /*: (LocalChange) => ?T */, predicate /*: (T) => boolean */, remove /*: ?true */) /*: ?T */ {
-  for (let i = 0; i < changes.length; i++) {
-    const anyChange = changes[i]
-    const rightTypeChange /*: ?T */ = maybeRightType(anyChange)
-    if (rightTypeChange != null && predicate(rightTypeChange)) {
-      if (remove) changes.splice(i, 1)
-      return rightTypeChange
-    }
-  }
-}
-
 function isChildMove (a /*: LocalChange */, b /*: LocalChange */) /*: boolean %checks */ {
```

Comment:
This one is actually not used anymore too (was inlined in analysis), but I didn't remove it yet.